islutils/builders.h

```diff
@@ -89,6 +89,30 @@ ScheduleNodeBuilder set(std::vector<ScheduleNodeBuilder> &&children);
 
 ScheduleNodeBuilder subtree(isl::schedule_node node);
 
+template <typename T>
+std::vector<ScheduleNodeBuilder> sequenceTransform(T t) {
+  return {t};
+}
+
+template<typename T, class ...Args>
+std::vector<ScheduleNodeBuilder> sequenceTransform(T t, Args... args)
+{
+   std::vector<ScheduleNodeBuilder> vv = sequenceTransform(args...);
+   std::vector<ScheduleNodeBuilder> vv2 = {t};
+   vv2.insert(std::end(vv2), std::begin(vv), std::end(vv));;
```

Comment:
Consider `std::make_move_iterator` to avoid copying here, `vv` is not used afterwards anyway, but I don't trust the compiler to be smart enough and remove the copy here.